docs/source/en/_toctree.yml

```diff
@@ -3,8 +3,6 @@
     title: 🧨 Diffusers
   - local: quicktour
     title: Quicktour
-  - local: stable_diffusion
```

Comment:
No I would like to keep it there for now. It's a highly visited website and this would break the link + at the moment `diffusers` == `stable diffusion` still

Reply:
Sounds good, moved back to the Get Started!